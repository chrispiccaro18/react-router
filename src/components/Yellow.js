import React from 'react';

export default function Yellow() {
  const divStyle = {
    width: '500px',
    height: '500px',
    backgroundColor: 'yellow'
  };
  return <div style={divStyle}></div>;
}
