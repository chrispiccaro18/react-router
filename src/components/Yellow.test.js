import React from 'react';
import { shallow } from 'enzyme';
import Yellow from './Yellow';

describe('Yellow component', () => {
  it('renders Yellow', () => {
    const wrapper = shallow(<Yellow />);
    expect(wrapper).toMatchSnapshot();
  }); 
});
