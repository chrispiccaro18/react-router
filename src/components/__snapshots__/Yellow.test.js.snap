// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Yellow component renders Yellow 1`] = `
<div
  style={
    Object {
      "backgroundColor": "yellow",
      "height": "500px",
      "width": "500px",
    }
  }
/>
`;
